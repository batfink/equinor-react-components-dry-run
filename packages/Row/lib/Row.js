import styled from 'styled-components';

const Row = styled.div`
    display: flex;
    flex-wrap: wrap;
    padding: 0px 20px;
    margin: 0px 0px 10px 0px;
`;

export default Row;
