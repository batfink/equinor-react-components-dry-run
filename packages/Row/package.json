{
  "name": "row",
  "version": "0.0.1",
  "description": "Row",
  "keywords": [
    "Row",
    "Flex",
    "Horizontal"
  ],
  "author": "fortena <arnigeir@gmail.com>",
  "homepage": "https://github.com/batfink/equinor-react-components-dry-run#readme",
  "license": "ISC",
  "main": "lib/Row.js",
  "directories": {
    "lib": "lib",
    "test": "__tests__"
  },
  "files": [
    "lib"
  ],
  "repository": {
    "type": "git",
    "url": "git+https://github.com/batfink/equinor-react-components-dry-run.git"
  },
  "scripts": {
    "test": "echo \"Error: run tests from root\" && exit 1"
  },
  "bugs": {
    "url": "https://github.com/batfink/equinor-react-components-dry-run/issues"
  }
}
