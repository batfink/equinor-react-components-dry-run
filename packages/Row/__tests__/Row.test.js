import React from 'react';
import Row from '../lib/Row';
import renderer from 'react-test-renderer';

describe('Row', () => {
    it('should render a Row', () => {
        const tree = renderer.create(<Row />).toJSON();
        expect(tree).toMatchSnapshot();
    });

    it('should be a flexbox', () => {
        const tree = renderer.create(<Row />).toJSON();
        expect(tree).toHaveStyleRule('display', 'flex');
    });

    it('should wrap', () => {
        const tree = renderer.create(<Row />).toJSON();
        expect(tree).toHaveStyleRule('flex-wrap', 'wrap');
    });
});
