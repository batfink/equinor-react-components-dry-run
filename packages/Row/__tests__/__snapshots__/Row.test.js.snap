// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row should render a Row 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  padding: 0px 20px;
  margin: 0px 0px 10px 0px;
}

<div
  className="c0"
/>
`;
